docs/MongoDB.md

```diff
@@ -18,21 +18,27 @@ Here's an example of the GlobalSetup script
 
 ```js
 // setup.js
+const path = require('path');
+
+const fs = require('fs');
+
 const MongodbMemoryServer = require('mongodb-memory-server');
 
-const MONGO_DB_NAME = 'jest';
-const mongod = new MongodbMemoryServer.default({
-  instance: {
-    dbName: MONGO_DB_NAME,
-  },
-  binary: {
-    version: '3.2.19',
-  },
-});
+const globalConfigPath = path.join(__dirname, 'globalConfig.json');
```

Comment:
Is it really necessary to create a file in project folder? I noticed such approach with jest-puppeteer example, but at least it creates file in `/tmp`, and anyway,  assigning to `global` should work fine?

Reply:
assigning to `global` only works when tests are run in band, not across workers

Reply:
I _think_ we pass `config` to the setup (and if not, we can start doing it) where you an lookup `cacheDirectory`

Reply:
From the little digging I did we don't have access to config in the setup, atleast not in an easy way.

Reply:
Yeah, it's #5957